prov/efa/src/rxr/rxr.h

```diff
@@ -538,6 +539,22 @@ struct rxr_domain {
 	size_t cq_size;
 };
 
+/** @brief Information of a queued copy.
+ *
+ * This struct is used when receiving buffer is on device.
+ * Under such circumstance, batching a series copies to
+ * do them at the same time can avoid memory barriers between
+ * copies, and improve performance.
+ */
+struct rxr_queued_copy {
+	struct rxr_pkt_entry *pkt_entry;
+	char *data;
+	size_t data_size;
+	size_t data_offset;
+};
+
+#define RXR_EP_MAX_QUEUED_COPY (8)
```

Comment:
Why 8 is the chosen value here?

Reply:
Experiments show 8 yielded best performance.

Reply:
OK, thanks.